packages/core/admin/server/middlewares/data-transfer.js

```diff
@@ -12,8 +12,11 @@ module.exports = () => async (ctx, next) => {
   }
 
   if (!hasValidTokenSalt()) {
-    return ctx.notImplemented(
-      'The server configuration for data transfer is invalid. Please contact your server administrator.'
+    return ctx.badRequest(
```

Comment:
I would keep the notImplemented, since it's not a bad request (the caller isn't doing anything wrong). Otherwise go with a 500 internal server error if that doesn't sound good.

Reply:
done thanks